eng/common/internal/Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
this is in eng/common and will be overwritten by arcade updates

Reply:
Thanks